tsup.config.js

```diff
@@ -1,6 +1,6 @@
 import { defineConfig } from 'tsup'
 import { esbuildPluginFilePathExtensions } from 'esbuild-plugin-file-path-extensions'
-import packageJson from './package.json';
+import packageJson from './package.json' assert { type: "json" };
```

Comment:
whats this? and why required now?

Reply:
I faced an issue with an on pre-commit from Husky for the mustache package. Fortunately, I discovered a solution to resolve this issue. While I had to comment out the use of the mustache package temporarily, I have preserved this code for future reference to enable the automatic generation of the README file using mustache.